src/test/java/showcase/neodymium/tests/random/FixedRandomTest.java

```diff
@@ -14,6 +14,18 @@
 import io.qameta.allure.Description;
 import showcase.neodymium.tests.AbstractTest;
 
+/**
+ * This show case shows the use of the Neodymium.getRandom() method.<br>
+ * Random values are useful to run different test cases with different values.<br>
+ * In some cases it is necessary to be able to repeat a test with the same random value (e.g. debugging).<br>
+ * Here you can see how a fixed random value can be used.<br>
+ * <br>
+ * <b>REQUIRED CONFIGURATION:</b> <i>config/neodymium.properties</i>
+ * <ul>
+ * <li>A value for neodymium.context.random.initialValue must be set.</li>
```

Comment:
you can already state the expected value in the documentation  

`... must be set to 123456789.`